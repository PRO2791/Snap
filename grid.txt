go to x:0 y:0
point in direction 0
clear
pen down
