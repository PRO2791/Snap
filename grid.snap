go to x:0 y:0
point in direction 0
clear
pen down
move 120 steps
move -120 steps
change x by -25
move 120 steps
move -120 steps
change x by -25
move 120 steps
move -120 steps
change x by -25
move 120 steps
move -120 steps
change x by -25
move 120 steps
move -120 steps
change x by -25
move 120 steps
move -120 steps
change x by -25
go to x:0 y:0
move 120 steps
